src/Pages/DiscoverRoversPage/DiscoverRoversPage.test.tsx

```diff
@@ -0,0 +1,17 @@
+﻿import React from "react";
+import {render} from "@testing-library/react";
+import {DiscoverRoversPage} from "./DiscoverRoversPage";
+
+
+test('check if photocards render', () => {
+    
+    let discoverPage = render(<DiscoverRoversPage />);
```

Comment:
This page contains some links, which you won't be able to render outside of a Router.
You'll need to render this component inside of a router to make it work.

Reply:
Fixed the test Mike

Reply:
Not quite the right way though! :) 

We've now got 2 routers in the application which 
a) shouldn't be necessary
b) might confuse the application (I've got no idea how nested routers work!)

Instead I think we should render inside a router that we make in the test - ie
```jsx
const discoverPage = render(
    <Router>
        <DiscoverRoversPage />
    </Router>
);
```

Reply:
Oh, silly me. Moved Router tag in the test